gallery/plot_transforms_v2_e2e.py

```diff
@@ -0,0 +1,147 @@
+"""
+==================================================
+transforms v2: End-to-end object detection example
+==================================================
+"""
+
+import pathlib
+from collections import defaultdict
+
+import PIL.Image
+
+import torch
+import torch.utils.data
+
+import torchvision
+
+
+def show(sample):
+    import matplotlib.pyplot as plt
+
+    from torchvision.transforms.v2 import functional as F
+    from torchvision.utils import draw_bounding_boxes
+
+    image, target = sample
+    if isinstance(image, PIL.Image.Image):
+        image = F.to_image_tensor(image)
+    image = F.convert_dtype(image, torch.uint8)
+    annotated_image = draw_bounding_boxes(image, target["boxes"], colors="yellow", width=3)
+
+    fig, ax = plt.subplots()
+    ax.imshow(annotated_image.permute(1, 2, 0).numpy())
+    ax.set(xticklabels=[], yticklabels=[], xticks=[], yticks=[])
+    fig.tight_layout()
+
+    fig.show()
+
+
+########################################################################################################################
+# To use ``transforms.v2``, you can use the stable datasets and models and just have import transforms differently
```

Comment:
Not sure if this was missed before but this sentences looks slightly weird. Below is one of many alternative suggestions

```suggestion
# To use ``transforms.v2``, you can use the stable datasets and models and you just need to import the transforms differently
```